cli/js/lib.deno.ns.d.ts

```diff
@@ -2153,23 +2212,25 @@ declare namespace Deno {
    */
   export function metrics(): Metrics;
 
-  /** **UNSTABLE**: reconsider representation. */
   interface ResourceMap {
+    // TODO(ry) can we change the "string" to "any" so we can extend this in the
+    // future.
     [rid: number]: string;
   }
```

Comment:
Makes no sense to lock ourselves into `[rid: number]` as we might change `rid` to something else (UUID? #4186) - I'm gonna change return type of `Deno.resources()` to `any`

Reply:
Well then you also need to change `rid` in all other places that reference it: `read`, `readSync`, `write`, `writeSync`, `seek`, `seekSync`, `close`, `File` and probably a lot more places.